src/DocusignBundle.php

```diff
@@ -20,6 +20,8 @@
 
 final class DocusignBundle extends Bundle
 {
+    public const TRANSLATION_DOMAIN = 'docusign_bundle';
```

Comment:
Should it be moved to `TranslatorAwareInterface` instead?

Reply:
At the moment I feel it like it's the right place...
When overriding from a symfony project perspective, it kinda makes sense that it come from the bundle.